The svn co as-is not yet usable, so we use SuSE's patch to get things done.
--- knx-0.1.orig/knx/src/knxconnection.cpp
+++ knx-0.1/knx/src/knxconnection.cpp
@@ -44,7 +44,7 @@
 KProcess *KNXConnection::SSHData::newProcess(QObject *parent) {
 	if (host.isEmpty() || user.isEmpty() || port.isEmpty()) return 0;
 		KProcess *proc=new KProcess(parent);
-		(*proc)<<"ssh" << "-i" <<"/usr/NX/share/client.id_dsa.key" << (user+"@"+host)<<"-p"<<port;
+		(*proc)<<"ssh" << "-i" <<"/usr/share/knx/client.id_dsa.key" << (user+"@"+host)<<"-p"<<port;
 		return proc;
 	}
 
@@ -99,7 +99,7 @@
 	kdDebug()<<"Trying to start ssh"<<endl;
 	m_sshProcess=m_sshData.newProcess(this);
 	if (!m_sshProcess) return false;
-	//(*m_sshProcess)<<"ssh" << "-i" <<"/usr/NX/share/client.id_dsa.key" << m_sshData.connectionString());
+	//(*m_sshProcess)<<"ssh" << "-i" <<"/usr/share/knx/client.id_dsa.key" << m_sshData.connectionString());
 	connect(m_sshProcess,SIGNAL(receivedStdout(KProcess *, char *, int)),
 		this,SLOT(receivedFromSSH(KProcess*, char*,int)));
 	connect(m_sshProcess,SIGNAL(wroteStdin(KProcess *)),this,SLOT(dataHasBeenWrittenToSSH()));
--- knx-0.1.orig/Makefile.am.in
+++ knx-0.1/Makefile.am.in
@@ -2,31 +2,6 @@
 ## (C) 1997 Stephan Kulow
 
 AUTOMAKE_OPTIONS = foreign 
-DISTCLEANFILES = inst-apps
 
-
-install-data-local:
-	@echo ""
-	@echo ""
-	@echo "*************** Important *************************"
-	@echo ""
-	@echo "This module contains unreleased software."
-	@echo ""
-	@echo "The software may compile and work, but it may just"
-	@echo "as well neither compile nor work."
-	@echo ""
-	@echo "****************************************************"
-	@echo ""
-
-$(top_srcdir)/acinclude.m4: $(top_srcdir)/admin/acinclude.m4.in $(top_srcdir)/admin/libtool.m4.in
-	@cd $(top_srcdir) && cat admin/acinclude.m4.in admin/libtool.m4.in > acinclude.m4 
-
-$(top_srcdir)/configure.in: $(top_srcdir)/configure.files $(shell test -f $(top_srcdir)/configure.files && cat $(top_srcdir)/configure.files)
-	cd $(top_srcdir) && $(MAKE) -f admin/Makefile.common configure.in
-$(top_srcdir)/subdirs:
-	cd $(top_srcdir) && $(MAKE) -f admin/Makefile.common subdirs
-$(top_srcdir)/configure.files:
-	cd $(top_srcdir) && $(MAKE) -f admin/Makefile.common configure.files
-
-MAINTAINERCLEANFILES = subdirs configure.in acinclude.m4 SUBDIRS configure.files
+include admin/deps.am
 
